plugins/modules/ilo_redfish_config.py

```diff
@@ -94,23 +242,197 @@
       attribute_name: StaticNTPServers
       attribute_value: X.X.X.X
 
+  - name: Create logical drive
+    ilo_redfish_config:
```

Comment:
Module name in examples must always be FQCN.

Reply:
Updated as suggested

Reply:
Resolving this conversation, thanks!